code/game/atoms.dm

```diff
@@ -81,6 +81,10 @@
 		reagents = null
 	invisibility = 101
 	return ..()
+	
+//Hook for running code when a dir change occurs
+/atom/proc/setDir(newdir)

```

Comment:
Ironically only about four files use set_dir (aka almost none). Considering the amount of stuff we port from /tg/, I think for the easy of -stealing- copying code we should use their proc/variable names.